### ISSUE TITLE
How to evaluate the training result of infoGan when inputting numerical training data?

### ISSUE THREAD

**Issue by PaperStrange** [PaperStrange/trick_infoGan#1]
By commenting the convolution-related layers and changing the input dimension, the infoGan could be trained on numerical data at now. One issue occurs that the loss, no matter the D loss or the Q loss oscillated obviously during the training procedure. As i know, the overfitting situation may be difficult to detect for the training of Gan network, maybe there is other standards available?
